requirements/optional.txt

```diff
@@ -5,3 +5,5 @@ ninja
 tensorboard
 imagesize==1.2.0
 PyYAML==5.3.1
+sty==1.0.0b12
```

Comment:
Just out of curiosity, why have you chosen this package?

Reply:
don't know how to draw palette using RGB values in terminal in other way. Maybe you know?

Reply:
No, I don't. I use termcolor, though not sure if it's able to handle arbitrary truecolor values.

Reply:
termcolor supports only several (less than 10) predefined colors